rootfs/etc/haproxy/template/haproxy.tmpl

```diff
@@ -127,7 +127,7 @@ frontend httpfront
 ###### HTTPS frontend (tcp mode)
 ######
 frontend httpsfront
-    bind *:443
+    bind *:443{{ if $cfg.UseProxyProtocol }} accept-proxy{{ end }}
```

Comment:
Do you see any problem on your deployment if `accept-proxy` is added to the stats binding after the implementation of a proper health-check url as described on #21 ? I have only bare metal deployments, won't use this feature myself but want to see a consistent configuration.